### PR TITLE
Improvements based on feedback

### PR DESCRIPTION
**Pull request by emcarru**

### CODE REVIEW

File: content/guide/components/server-side-logic.md [hunk 1]
```diff
@@ -73,10 +74,12 @@ add_action('wp_enqueue_scripts', function ()
...
+This will look for `flynt-starter-theme/dist/vendor/slick.js` and `flynt-starter-theme/dist/vendor/slick.css` and enqueue the files if found.
 
-## Using Filters
-The two main filters you will use inside `functions.php` are `Flynt/addComponentData` and `Flynt/dynamicSubcomponents`.
+**Never add files manually to the `dist` folder of the theme.** To compile dependencies into the `dist/vendor` folder of the theme, see the [section on copying vendor files here](/guide/components/client-side-scripts/#add-third-party-dependencies).
+
+## Add and Modify Component Data with Filters
+[Filters are provided by WordPress](https://codex.wordpress.org/Plugin_API) to allow a plugin to 'hook into' the rest of WordPress and call functions at specific times. [Flynt Core](/guide/core/) defines two filters most often used inside `functions.php` for the purpose of adding component data and [areas](/guide/components/what-is-component#what-is-an-area). These are `Flynt/addComponentData` and `Flynt/dynamicSubcomponents`.
```
Comment (Qakulukiam) [inline content/guide/components/server-side-logic.md]: Maybe consider linking the two functions to the API docs?

File: content/guide/components/server-side-logic.md [hunk 2]
```diff
@@ -183,3 +188,5 @@ add_filter('Flynt/addComponentData?name=ExampleComponentName', function ($data)
     return $data;
 });
 ```
+
+[Learn more about using utils in the Utils section](/guide/utils/using-utils/).
```
Comment (Qakulukiam) [inline content/guide/components/server-side-logic.md]: Just wondering, why is one the utils uppercase and the other isn't?

File: content/guide/components/what-is-component.md [hunk 3]
```diff
@@ -28,5 +28,113 @@ A component is a self-contained building-block. As such, each component is kept
...
+<html class="flyntComponent {{ body_class }}" lang="{{ site.language }}" dir="{{ dir }}" is="flynt-document-default">
+  <head><!--...--></head>
+  <body role="document">
+    {{ area('layout') }}
+    {{ wp_footer }}
+  </body>
+</html>
+```
+
+The `area` helper function is created by the [TimberLoader](https://github.com/flyntwp/flynt-starter-theme/blob/master/Features/TimberLoader/README.md) feature using the `renderComponent` filter provided by the [Flynt Core plugin](/guide/core/).
```
Comment (Qakulukiam) [inline content/guide/components/what-is-component.md]: "Flynt Core plugin" here, just "Flynt Core" somewhere above in this PR..

File: content/guide/components/what-is-component.md [hunk 4]
```diff
@@ -28,5 +28,113 @@ A component is a self-contained building-block. As such, each component is kept
...
+
+## Organising Components into Categories
```
Comment (Qakulukiam) [inline content/guide/components/what-is-component.md]: Organizing 🇺🇸

File: content/guide/components/what-is-component.md [hunk 5]
```diff
@@ -28,5 +28,113 @@ A component is a self-contained building-block. As such, each component is kept
...
+
+Components are always categorised according to their predominant use case. This helps to clarify the purpose of a component and in quickly identifying an appropriate component to re-use. The available categores are as follows:
```
Comment (Qakulukiam) [inline content/guide/components/what-is-component.md]: categores

File: content/guide/components/what-is-component.md [hunk 6]
```diff
@@ -28,5 +28,113 @@ A component is a self-contained building-block. As such, each component is kept
...
+      <strong>Accordion</strong><br>
+      For collapsable content, e.g. FAQs.
+    </li>
+    <li>
+      <strong>Audio</strong><br>
+      Styled audio players.
+    </li>
+    <li>
+      <strong>Block</strong><br>
+      The fallback for full width components. Use only when there is no better, more specific category.
```
Comment (Qakulukiam) [inline content/guide/components/what-is-component.md]: doesn't have to be full width, right?

File: content/guide/components/what-is-component.md [hunk 7]
```diff
@@ -28,5 +28,113 @@ A component is a self-contained building-block. As such, each component is kept
...
+    </li>
+    <li>
+      <strong>Box</strong><br>
+      A content box wrapped in or on top of something else (like the css box model).
+    </li>
+    <li>
+      <strong>Cols</strong><br>
+      Its elements will never flow into a new row.
```
Comment (Qakulukiam) [inline content/guide/components/what-is-component.md]: Please make sure you are actually using the correct categories, some of these don't exist anymore. (Some of them never have?)

File: content/guide/components/what-is-component.md (hunk 5 above)
Comment (Qakulukiam) [inline content/guide/components/what-is-component.md]: categorized 🇺🇸

Comment (Qakulukiam) [inline content/guide/components/what-is-component.md]: It's also important to mention that categorizing means prefixing the component with the category.

File: content/guide/utils/using-utils.md [hunk 8]
```diff
@@ -26,4 +26,6 @@ use Flynt\Utils\Asset;
 $image = Asset::requireUrl('Components/ListPosts/assets/image.jpg');
 ```
 
+All Utils are loaded in `lib/Init.php`, and are availaible for use in any file loaded after this point in execution. Any PHP file loaded before this point will need to take care of loading any Util itself.
```
Comment (Qakulukiam) [inline content/guide/utils/using-utils.md]: Using "any" a lot.

File: content/guide/components/what-is-component.md [hunk 9]
```diff
@@ -48,33 +48,25 @@ For example, the `DocumentDefault` component renders the `layout` area in `Docum
...
     </li>
     <li>
-      <strong>Box</strong><br>
-      A content box wrapped in or on top of something else (like the css box model).
-    </li>
-    <li>
-      <strong>Cols</strong><br>
-      Its elements will never flow into a new row.
+      <strong>Calendar</strong><br>
+      For a component where the mean feature is a calendar.
```
Comment (Qakulukiam) [inline content/guide/components/what-is-component.md]: why so mean? I'm guessing this should be main?

File: content/guide/utils/using-utils.md [hunk 10]
```diff
@@ -26,6 +26,6 @@ use Flynt\Utils\Asset;
 $image = Asset::requireUrl('Components/ListPosts/assets/image.jpg');
 ```
 
-All Utils are loaded in `lib/Init.php`, and are availaible for use in any file loaded after this point in execution. Any PHP file loaded before this point will need to take care of loading any Util itself.
+All Utils are loaded in `lib/Init.php`, and are availaible for use in all files loaded after this point in execution. Any PHP file loaded before this point will need to take care of loading the Util itself.
```
Comment (Qakulukiam) [inline content/guide/utils/using-utils.md]: availaible

File: content/guide/components/what-is-component.md [hunk 11]
```diff
@@ -48,33 +48,25 @@ For example, the `DocumentDefault` component renders the `layout` area in `Docum
...
-The `area` helper function is created by the [TimberLoader](https://github.com/flyntwp/flynt-starter-theme/blob/master/Features/TimberLoader/README.md) feature using the `renderComponent` filter provided by the [Flynt Core plugin](/guide/core/).
+The `area` helper function is created by the [TimberLoader](https://github.com/flyntwp/flynt-starter-theme/blob/master/Features/TimberLoader/README.md) feature using the `renderComponent` filter provided by [Flynt Core](/guide/core/).
 
-## Organising Components into Categories
-
-Components are always categorised according to their predominant use case. This helps to clarify the purpose of a component and in quickly identifying an appropriate component to re-use. The available categores are as follows:
+## Organizing Components into Categories
 
+Components are always categorized according to their predominant use case. This helps to clarify the purpose of a component and in quickly identifying an appropriate component to re-use. The name of every component is prefixed with one of the following categories:
+- Table
```
Comment (Qakulukiam) [inline content/guide/components/what-is-component.md]: what is this for?

File: content/guide/components/what-is-component.md (hunk 9 above)
Comment (emcarru) [inline content/guide/components/what-is-component.md]: Wow. Yep. Thanks

**Pull request merged**